modules/ocf_desktop/manifests/stats.pp

```diff
@@ -27,9 +27,10 @@
   }
 
   cron { 'labstats':
-    ensure  => present,
-    command => '/opt/stats/update.sh > /dev/null',
-    user    => 'ocfstats',
-    minute  => '*';
+    ensure      => present,
+    environment => 'PATH=/usr/local/sbin:/usr/local/bin:/bin:/usr/sbin:/usr/bin:/opt/puppetlabs/bin',
```

Comment:
Could we just add `/opt/puppetlabs/bin` in the global cron path to make this cleaner?

https://github.com/ocf/puppet/blob/26109d27c9c19d1b683c9a32af428a0a01ebe51c/manifests/site.pp#L38-L48